python/cudf/cudf/comm/serialize.py

```diff
@@ -44,7 +44,7 @@ def deserialize_cudf_object(header, frames):
                     # some frames are empty -- meta/empty partitions/etc
                     if len(f) > 0:
                         assert hasattr(f, "__cuda_array_interface__")
-            if header["serializer"] == "dask":
+            elif header["serializer"] == "dask":
```

Comment:
Moving discussion to hear. I understand that this should be `elif`, but how could both conditions evaluate to True here regardless?

Reply:
It wouldn't be possible.

Reply:
@kkraus14 are you suggesting this should be `else` and not `elif` ?

Reply:
No, I wasn't aware that `header['serializer']` could be a list of elements which would allow both conditions to result to `True`.

Reply:
I think that is incorrect. `header["serializer"]` should only be a single `str` with a serializer named.

Ben was mentioning that multiple serializers could be used to transform the data (and its various components), which is different from the value seen here.

Reply:
@jakirkham So then is this a fix for something or just preventing an additional conditional check?

Reply:
Just preventing an additional check. Something I stumbled across recently when looking at the serialization code.